<?xml version="1.0"?>

<launch>
    
    <node pkg="steering_guidance" type="steering_guidance_node" name="steering_guidance_node" output="screen">
        
        <param name="sub_topic" value="/usb_cam/image_raw"/>
        <param name="pub_topic" value="/steering_guidance_image"/>
        
        <param name="length_b" value="1.6"/>
        <param name="length_l" value="2.3"/>
        <param name="length_offset" value="1.0"/>
        <param name="height_of_camera" value="1.8"/>
        
        <param name="cam_fx" value="581.0"/>
        <param name="cam_fy" value="604.0"/>
        <param name="cam_u0" value="605.0"/>
        <param name="cam_v0" value="332.0"/>
        <param name="image_width" value="1280"/>
        <param name="image_height" value="720"/>
        
        <param name="num_points" value="1000"/>
        <param name="min_distance" value="2"/>
        <param name="max_distance" value="15"/>
        
        <rosparam param="steering_angle" > 0 </rosparam>
        
    </node>

</launch>
